app/views/root/local_transaction.html.erb

```diff
@@ -58,3 +58,17 @@
     </div>
   </section>
 <% end %>
+
+<% if @postcode_error %>
+  <script type="text/javascript">
+    //<![CDATA[
+      GOVUK.analytics.trackEvent(
+        "userAlerts:LocalTransaction",
+        "postcodeErrorShown:<%= @postcode_error %>",
```

Comment:
Both points are entirely correct according to past examples and current guidance. But there's a project under way to revise our approach to GA events and this is an early example.

The "local_transaction" detail is indeed duplicated in the "format" dimension. Duplication here was 1) to surface it in standard GA reports for users who do not use custom reporting 2) to allow splitting potential "userAlerts" at a high level in reports and so leave more scope for other data at  lower levels.

Moving to camelCase is intended to compress the string slightly because we're adding details here and want to save space in reports.

I've reviewed the comment with Paul Cronk and Peter Jordan. We took the comment on board but would prefer to stick with the new format please.